examples/index.html

```diff
@@ -43,6 +43,8 @@
       <!-- To enable auto-select (select first on Enter), use data-autoselect="true" -->
       <auto-complete src="/demo" for="items-popup" aria-labelledby="robots-label" data-autoselect="true">
         <input name="robot" type="text" aria-labelledby="robots-label" autofocus>
+        <!-- if a clear button is passed in, recommended to be *before* UL elements to avoid conflicting with their blur logic -->
+        <button id="robot-clear">x</button>
```

Comment:
may we have an example using the input `id`?

Reply:
For sure 👍